components/concept/engine/src/main/java/mozilla/components/concept/engine/Settings.kt

```diff
@@ -174,6 +175,11 @@ abstract class Settings {
      * Setting to force the ability to scale the content
      */
     open var forceUserScalableContent: Boolean by UnsupportedSetting()
+
+    /**
+     * Setting to control the download path.
+     */
+    open var downloadPath: String by UnsupportedSetting()
```

Comment:
I think will need add the setting to [DefaultSettings](https://github.com/mozilla-mobile/android-components/blob/16343bb7702b39a9ec04ca64e5cb2314d580b85a/components/concept/engine/src/main/java/mozilla/components/concept/engine/Settings.kt#L182) :)